.github/workflows/ci.yml

```diff
@@ -0,0 +1,121 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - develop
+      - backports/**
+  pull_request:
+    branches:
+      - develop
+      - backports/**
+    paths-ignore:
+      - '**.rst'
+      - '**.md'
+      - 'scripts/dev'
+  merge_group:
+    types: [checks_requested]
+
+concurrency:
+  group: ${{github.ref}}-${{github.event.pull_request.number || github.run_number}}-${{github.workflow}}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: CI
+    strategy:
+      matrix:
+        image: ['ubuntu-cuda', 'centos-rocm']
+        buildtype: ['debug', 'ndebug']
+        geometry: ['orange', 'vecgeom']
+        special: [null]
+        exclude:
+          - image: 'centos-rocm'
+            geometry: 'vecgeom'
+        include:
+          - image: 'centos-rocm'
+            buildtype: 'asan'
+            geometry: 'orange'
+          - image: 'centos-rocm'
+            buildtype: 'debug'
+            geometry: 'orange'
+            special: 'float'
+          - image: 'centos-rocm'
+            buildtype: 'reldeb'
+            geometry: 'geant4'
+          - image: 'ubuntu-cuda'
+            buildtype: 'reldeb'
+            geometry: 'vecgeom'
+    env:
+      ASAN_OPTIONS: "detect_leaks=0"
+        #      CCACHE_DIR: "${{github.workspace}}/.ccache"
+        #      CCACHE_MAXSIZE: "10G"
+      CELER_TEST_STRICT: 1
+      CELER_DISABLE_DEVICE: 1 # IMPORTANT
+      CMAKE_PRESET: >-
+        ${{matrix.buildtype}}-${{matrix.geometry}}${{matrix.special && '-' || ''}}${{matrix.special}}
+    runs-on: ubuntu-latest
+    container:
+      image: >-
+        docker.io/celeritas/${{
+             matrix.image == 'ubuntu-cuda' && 'ci-jammy-cuda11:2023-08-02'
+          || matrix.image == 'centos-rocm' && 'ci-centos7-rocm5:2022-12-14.2'
+          || null
+        }}
+      # See https://github.com/actions/checkout/issues/956
+      options: --user root
+    steps:
+      - name: Set up environment
+        run: |
+          . /etc/profile
+          echo "/opt/view/bin" >> $GITHUB_PATH
+          echo "CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}" >> $GITHUB_ENV
+      # NOTE: checkout must occur *after* setting up environment for git tags to work
+      # NOTE: checkout v4 fails: https://github.com/actions/checkout/issues/1487
+      # NOTE: depth must be enough to include the previous tag
+      - name: Check out Celeritas
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 255
+          fetch-tags: true
+      - name: Configure Celeritas
+        run: |
+          git config --global --add safe.directory ${PWD}
+          ln -fs scripts/cmake-presets/ci-${{matrix.image}}.json CMakeUserPresets.json
```

Comment:
This means that we don't (yet) support Windows build :)

Reply:
Our CI has never supported windows builds and it's running inside a docker container that's linux only. 😝 

This PR is really just to get us back in business to most of the capability we had before. I'd like to extend testing (see comment in the PR description) and if you want to add a windows build I'm a ll for it.